numpy/core/src/umath/loops_trigonometric.dispatch.c.src

```diff
@@ -2,19 +2,25 @@
  ** $maxopt baseline
  ** (avx2 fma3) avx512f
  ** vsx2 vsx3 vsx4
- ** neon_vfpv4
+ ** asimd
```

Comment:
```suggestion
 ** neon_vfpv4
```

`neon_vfpv4` already implies `asimd` on aarch64 since it's part of the hardware baseline so no need for this change.